app/controllers/api_controller/blueprints.rb

```diff
@@ -0,0 +1,28 @@
+class ApiController
+  module Blueprints
+    def create_resource_blueprints(_type, _id, data)
+      attributes = data.except("bundle")
```

Comment:
👍   In a separate PR, we could revisit the current create_* methods and eliminate the redundant id/href blacklist checks and replace with tests like you did here.  Thanks!!